pkg/kv/kvclient/kvcoord/dist_sender_rangefeed_test.go

```diff
@@ -260,71 +257,6 @@ func channelWaitWithTimeout(t *testing.T, ch chan struct{}) {
 		t.Fatal("test timed out")
 	}
 }
-func TestBiDirectionalRangefeedNotUsedUntilUpgradeFinalilzed(t *testing.T) {
```

Comment:
Why are we deleting this test? Also, can you move the delete into a separate commit?

Reply:
Sure, deleted in #111396

Reply:
This and the other test were deleted because they are no longer needed in the current branch (they were testing version gates that no longer exist).